src/MSIdentityScaffolding/README.md

```diff
@@ -1,7 +1,11 @@
 # dotnet-msidentity
 Command line tool that creates Microsoft identity platform applications in a tenant (AAD or B2C) and updates the configuration code of your ASP.NET Core applications (mvc, webapp, blazorwasm, blazorwasm hosted, blazorserver). The tool can also be used to update code from an existing AAD/AAD B2C application.
 
-## Installing/Uninstalling the tool
+## Installing/Uninstalling the release/prerelease tool 
```

Comment:
I think the previous title was better. Adding `release/prerelease` isn't adding value here. Later if we add instructions for both release & prerelease we will just do sub-headings with 3 `#` symbols.